content/manager/bootstrapping.md

```diff
@@ -165,6 +165,153 @@ Services:
+    [Openstack image upload instructions](http://docs.openstack.org/user-guide/dashboard_manage_images.html)
+    {{% /gsNote %}}
+
+ 1. Create an instance based on the image you've uploaded.
+
+    {{% gsNote %}}
+    Make sure you enable the relevant security settings in the instance's security group. Port `22` is required for `ssh` access, and ports `80` and `443` are required for HTTP(S) access.
+    {{% /gsNote %}}
+
+ 1. Make a note of the IP/hostname
+
+{{< gsHighlight  sh  >}}
+    $ CLOUDIFY_HOST={your-manager-public-ip}
+{{< /gsHighlight >}}
+
+
+## Web UI
+
+ 1. In your browser, navigate to http://{your-manager-public-ip}
+
+    {{% gsNote %}}
+    The manager VM may take some time to start up, so if it doesn't load immediately don't worry.
+    {{% /gsNote %}}
+
+    {{% gsNote %}}
+    If security is enabled, your browser will complain that the TLS/SSL certificate is not valid. Temporarily allow the connection: A new certificate will be generated as part of this setup process.
+    {{% /gsNote %}}
+
+ 2. If asked, authenticate using `cloudify` as both the username and password.
+
+ 3. Create a new deployment of the `CloudifySettings` blueprint:
+    ![CloudifySettings blueprint deploy]({{< img "manager/image-deploy-new-deployment.png" >}})
+
+ 5. Fill in the required fields:
+
+    | name                            | platform  | secure only | notes                                                                                                                                                                                                                                                                    |
+    | -----                           | -----     | -----       | -----                                                                                                                                                                                                                                                                    |
+    | user_ssh_key                    |           |             | This will be added to ~/.ssh/authorized_keys in addition to any key you specified when initializing the instance.                                                                                                                                                        |
+    | agents_security_group_name      |           |             | Security group to be generated and used for accessing agents on VMs. This must not alerady exist as the `install` workflow will create a new one                                                                                                                         |
+    | agents_keypair_name             |           |             | Keypair to be generated and used for accessing agents on VMs. This must not alerady exist as the `install` workflow will create a new one.                                                                                                                               |
```

Comment:
No it's mine :( I rewrote most of them.
Switched spell on for markdown files in vim now... :)